packages/core/src/impl-store.ts

```diff
@@ -180,11 +205,18 @@ async function checkForAddressClash(
       debug('deleting a previous deployment at address', updated.address);
       clash.set(undefined);
     } else {
-      throw new Error(
-        `The following deployment clashes with an existing one at ${updated.address}\n\n` +
-          JSON.stringify(updated, null, 2) +
-          `\n\n`,
-      );
+      const existing = clash.get();
+      // it's a clash if there is no deployment id or if deployment ids don't match
```

Comment:
Can you describe for me in what scenario we would see this? Finding an existing entry with the same deployment id.

Reply:
When calling resumeOrDeploy before checkForAddressClash, resumeOrDeploy may detect that the transaction hash changed for the deployment id.  In this case, it returns an updated entry for the deployment.

We then call checkForAddressClash with this updated entry.  The address and deployment ids match, but the transaction hash changed, so we don't want this to be treated as a clash.  We just want to update the manifest.

I'll add a test for this.

Reply:
Added test in https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/763/commits/a9b5147abdd3256d974ff2a803ae5302232c0903